docs/api/router-dashboard.md

```diff
@@ -106,6 +175,45 @@ This file documents the dashboard API found in Rita client.
 
 `curl 127.0.0.1:4877/exits`
 
+---
+## /exits/sync
+
+- URL: `<rita ip>:<rita_dashboard_port>/exits/sync'
+- Comment: Adds exits from under `url` remote HTTP host to exit list;
+  conflicting entries are overwritten by remote list contents
+- Method: `GET`
+- URL Params: `None`
+- Data Params:
+```
+{
+  "url": "https://wherever.the/json/list/is.json"
+}
+```
+- Success Response:
+  - Code: 200 OK
+  - Contents: Updated exit list (see POST `/exits` for example)
+- Error Response: `400 Bad Request` for unparsable response JSON, `500 Internal Server
+  Error` when the request itself fails for whatever reason
+- Error Contents:
+  - `400 Bad Request` when the JSON is unparsable
+```json
+{
+  "error": "<description>"
+}
+```
+  - `500 Internal Server Error` when the request fails
+```json
+{
+  "error": "<description>",
+  "rust_error": "<stringified_rust_error>"
+}```
+
+- Sample Call:
+
+`curl 127.0.0.1:4877/exits/borked/reset -H "Content-Type:
```

Comment:
Accidentally wrapped in the middle of command?

Reply:
yep